wallet/wallet.go

```diff
@@ -2709,17 +2709,37 @@ func (w *Wallet) ListUnspent(minconf, maxconf int32,
 	return results, err
 }
 
+type ListLeasedOutputResult struct {
```

Comment:
Should we call this `LockedOutputDetail` instead? Also, some Godoc comment would be good.

Reply:
I named it `ListLeasedOutputResult` because it is more consistent with return types of other functions. For eg. [AccountBalanceResult](https://github.com/btcsuite/btcwallet/blob/50ea8d1125561279f9c9af1e46bea16ca414b4a5/wallet/wallet.go#L2377) , [ListUnspentResult](https://github.com/btcsuite/btcwallet/blob/50ea8d1125561279f9c9af1e46bea16ca414b4a5/wallet/wallet.go#L2583), [AccountBalanceResult](https://github.com/btcsuite/btcwallet/blob/50ea8d1125561279f9c9af1e46bea16ca414b4a5/wallet/wallet.go#L2461). `LockedOutputDetail` also seems good  to me though.